src/adv.py

```diff
@@ -60,3 +67,38 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
```

Comment:
This is a massive jump in commits, you went from zero to functioning app. 

Make MORE commits, showcase each working code